src/main/java/racingcar/Winners.java

```diff
@@ -0,0 +1,32 @@
+package racingcar;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Winners {
+    public static List<Car> findWinners(List<Car> cars) {
+        return findWinners(cars, getMaxPosition(cars));
+    }
+
+    private static List<Car> findWinners(List<Car> cars, int maxPosition) {
+        List<Car> winners = new ArrayList<>();
+        for (Car car : cars) {
+            addIfMaxPosition(maxPosition, car, winners);
+        }
+        return winners;
```

Comment:
Java의 stream을 활용해보셔도 좋을 것 같습니다.
```suggestion
        return cars.stream()
            .filter(car -> car.getCurrentPosition() == maxPosition)
            .collect(Collectors.toList());
```